src/Sentry/Laravel/Tracing/EventHandler.php

```diff
@@ -108,6 +115,8 @@ public function __construct(array $config, BacktraceHelper $backtraceHelper)
         $this->traceSqlQueries = ($config['sql_queries'] ?? true) === true;
         $this->traceSqlQueryOrigins = ($config['sql_origin'] ?? true) === true;
 
+        $this->traceHttpClientRequests = ($config['http_client_requests'] ?? true) === true;
```

Comment:
Opted for enabled by default, to not break for people that already use it without updating their config.